src/env.rs

```diff
@@ -197,6 +221,13 @@ impl Source for Environment {
             if !separator.is_empty() {
                 key = key.replace(separator, ".");
             }
+            if let Some(translate_key) = translate_key {
+                match translate_key {
+                    TranslationType::Kebab => {
+                        key = key.replace("_", "-");
+                    },
+                }
+            }
```

Comment:
```suggestion
            if let Some(TranslationType::Kebab) = translate_key {
                key = key.replace("_", "-");
            }
```

But if we add more types then this is not possible of course. :laughing: